src/docs/user/references/JobPlannerServiceReference.adoc

```diff
@@ -49,26 +53,50 @@ Then, there are 3 ways to visualize the recurrence on the Execution Planning pan
+In addition to the Execution Planning panel, the Gantt chart allows you to see the past job history, the current one being executed, as well as the future ones, all in a comprehensive interactive view. In this view, you will easily see the potential difference between the `Planned Submission time` and the `Actual Start Time` of the Jobs. You will also get estimations of the `Finished Time`, taking into account the `Actual Start Time`.
+Moreover, you can visualize the Job that stayed PENDING for some time (in Yellow), as well as the Jobs that had issues and got KILLED, CANCELLED, or FAILED (in red).
 
 The planned jobs (in orange) are on the row just above the actual jobs so that you can easily compare what was planned and what actually executed. You can use the legend button for a better understanding of the colors.
 
 image::calendar_planning_gantt_legend.png[align="center"]
 
-If a job has been PENDING before starting, depending on the scale you have selected, you can see a yellow part on the displayed job. This corresponds to the time the job was PENDING and helps to highlight a resource problem.
-
-The length of the displayed planned jobs can be :
+The length of the displayed planned jobs can be:
 
     - the expected execution time given by users in the workflow with the Generic Information: `JOB_EXEC_TIME`
     - if the workflow doesn't have any `JOB_EXEC_TIME` Generic Information, it will be the average execution time computed from the past association executions
-    - if the workflow doesn't have any `JOB_EXEC_TIME` Generic Information and has never been executed with the association, it will be a default 15 minutes duration.
+    - if the workflow doesn't have any `JOB_EXEC_TIME` Generic Information and has never been executed with the association, it will be a default 15 minutes duration
+
+image::calendar_planning_gantt_tooltip.png[align="center"]
+
+If a job has been PENDING before starting, depending on the scale you have selected, you can see a yellow part on the displayed job. This corresponds to the time the job was PENDING and helps to highlight a resource problem.
+
+You can see more information jobs in the tooltips, when passing your mouse over a specific job:
+
+    - `Workflow` and `bucket name`
+    - `Job ID`: the ID given by the scheduler when a job has been submitted
+    - `Status`: the status of the job when the Gantt has been generated
+    - `User estimated execution duration`: Generic Information `JOB_EXEC_TIME` given by users in the workflow. This information doesn't appear if the Generic Information is not defined in the workflow.
+    - `Submission time`: the time when the job had been submitted by the job planner.
```

Comment:
sometimes there dots in the end of sentences sometimes there are nothing

Reply:
It's when I start a new sentence (for example : This information...). I removed one